blocklib/fec/lib/reed-solomon/README

```diff
@@ -0,0 +1,2 @@
+This code is from http://people.qualcomm.com/karn/code/fec
+It is based on reed-soloman-3.1.1 (1 Jan 2002).
```

Comment:
Maybe this would fit better as subprojects/reed-solomon?

Reply:
Perhaps - this was a straight port from GR.  I'll leave it as is for now, but open to a later PR